### PR TITLE
Fill in the missing classes!

### PR DESCRIPTION
**Pull request by mounderfod**
There are only 4 classes unnamed in Yarn right now. This fills in three of them:
- `ShulkerLookControl` in `ShulkerEntity` (many other entities have the exact same subclass)
- `PlaneHelper` in `Camera` (a math**s** helper for managing planes)
- `HoverableButtonWidget` in `BeaconScreen` (this is a janky name and I'm willing to change it, it just reflects what the subclass does)

Feedback needed, these are janky as hell and could do with refining; I just wanted to get the change moving!

### CODE REVIEW

File: mappings/net/minecraft/client/render/Camera.mapping [hunk 1]
```diff
@@ -48,3 +48,15 @@ CLASS net/minecraft/class_4184 net/minecraft/client/render/Camera
 	METHOD method_19337 reset ()V
 	METHOD method_23767 getRotation ()Lnet/minecraft/class_1158;
 	METHOD method_35689 getDiagonalPlane ()Lnet/minecraft/class_1160;
+	METHOD method_36425 getMathHelper ()Lnet/minecraft/class_4184$class_6355;
```
Comment (IotaBread) [inline mappings/net/minecraft/client/render/Camera.mapping]:
```suggestion
	METHOD method_36425 getPlaneHelper ()Lnet/minecraft/class_4184$class_6355;
```

Comment (mounderfod) [reply mappings/net/minecraft/client/render/Camera.mapping]: That one got away, haha, thanks 👍

**Review by enbrain — COMMENTED** [review]
`class_6355` looks more like just a plane or some specialized plane. I think 4 methods in `class_6355` are the 4 corners of the plane.

**Comment by mounderfod** [comment]
> `class_6355` looks more like just a plane or some specialized plane. I think 4 methods in `class_6355` are the 4 corners of the plane.

Interesting... happy to rename if needed

**Comment by liach** [comment]
see #2513 for shulker look control too. and plane helper is definitely a wrong name.

**Comment by mounderfod** [comment]
Apparently my PR is redundant because someone made a separate one altogether instead of reviewing and then latert adding javadoc (#2532), also #2513 does the look control.

**Comment by liach** [comment]
No? You can still keep your change for the look control and beacon widget. I said plane control was definitely a wrong name and only request that to be either removed or fixed. In addition, I have been looking at the camera stuff before you submitted this pr (because I know how to look at the stats in enigma, too)

File: mappings/net/minecraft/client/render/Camera.mapping [hunk 2]
```diff
@@ -48,3 +48,15 @@ CLASS net/minecraft/class_4184 net/minecraft/client/render/Camera
...
+		FIELD field_33622 horizontalPlane Lnet/minecraft/class_243;
+		FIELD field_33623 verticalPlane Lnet/minecraft/class_243;
+		FIELD field_33624 diagonalPlane Lnet/minecraft/class_243;
+		METHOD <init> (Lnet/minecraft/class_243;Lnet/minecraft/class_243;Lnet/minecraft/class_243;)V
+			ARG 1 horizontalPlane
+			ARG 2 verticalPlane
+			ARG 3 diagonalPlane
+		METHOD method_36427 (FF)Lnet/minecraft/class_243;
+			ARG 1 verticalMultiplier
+			ARG 2 diagonalMultiplier
```
Comment (liach) [inline mappings/net/minecraft/client/render/Camera.mapping:62]:
```suggestion
```
Can probably remove this for now

**Comment by liach** [comment]
@redcreeper14385 If you can remove the camera changes, I think this is quite good. I can fix merge conflict for the shulker one later

**Comment by mounderfod** [comment]
I'd rather make a separate PR at this point - besides I'm not particularly bothered about this PR anymore.

**Pull request closed without merging**